docs/apps/trinity.md

```diff
@@ -71,6 +79,8 @@ More information about running batch jobs, can be found from the chapter three o
 
 Please check the Trinity site to get hints for estimating the required resources,
 
+
+
  
 ## Using autoTrinotate in Taito
```

Comment:
I would like to remove this Taito specific part, is that ok?

Reply:
Haluaisin pitää näkyvillä maininnan siitä että autoTrinotate on saatavilla Taidossa.
Sitten kun ohjelma on myös puhdissa  niin  mainita voidaan poistaa. 

Tämä ei siis ole "näin käytät Trinityä Puhdissa" -ohje vaan  "Näin käytät Trinityä CSC:llä" -ohje.